apps/portals/src/configurations/elportal/routesConfig.tsx

```diff
@@ -184,28 +184,6 @@ const routes: GenericRoute[] = [
   {
     path: 'Explore',
     routes: [
-      {
-        path: ':slug/',
-        hideRouteFromNavbar: true,
-        exact: true,
-        synapseConfigArray: [
-          {
-            name: 'RouteControlWrapper',
```

Comment:
RouteControlWrapper should appear below OrientationBanner, so change the route order so it comes after